_format/1.1/index.md

```diff
@@ -2371,6 +2393,23 @@ An error object **MAY** have the following members:
     exists; if it doesn't, the client **SHOULD** simply ignore the pointer.
   * `parameter`: a string indicating which URI query parameter caused
     the error.
+  * <a id="error-objects-source-missing"></a>`missing`: an object indicating
+    that the error was caused by a missing profile, JSON object key (or array 
+    index), or URI query parameter. If present, this object **MUST** contain 
+    exactly one of the following keys:
+      * `extension`: the URI of a missing profile. This key **SHOULD** be
+        used with the `pointer` member to indicate which value in the request
+        document is missing the profile.
+      * `key`: the name of a missing key in a JSON object, or a missing index in 
+        a JSON array, expressed as a string. This key **SHOULD** be used with 
+        the `pointer` member to indicate which object/array in the request 
+        document is missing the key.
```

Comment:
Even if server-required profiles doesn't make it into 1.1, I think this is a worthwhile addition, paired with the language added in https://github.com/json-api/json-api/commit/e75cec15f0e663a9676fcc746b0801a773394647 to clarify that the pointer must reference an element in the original request. In the case of a missing key, this should be paired with some other data point (e.g., this element) to tell the client exactly which data was missing.